examples/fetch/src/post.rs

```diff
@@ -0,0 +1,66 @@
+//! Fetch POST example.
+
+use seed::{prelude::*, *};
+
+#[derive(serde::Serialize, Default)]
+pub struct Form {
+    name: String,
+}
+
+#[derive(Default)]
+pub struct Model {
+    form: Form,
+}
+
+pub enum Msg {
+    NameChanged(String),
+    Submit,
+    Submited,
+}
+
+pub fn update(msg: Msg, model: &mut Model, orders: &mut impl Orders<Msg>) {
+    match msg {
+        Msg::NameChanged(name) => model.form.name = name,
+        Msg::Submit => {
+            orders.skip(); // No need to rerender
+
+            let token = "YWxhZGRpbjpvcGVuc2VzYW1l";
+            // Created outside async block for lifetime reasons.
+            let request = Request::new("/")
+                .method(Method::Post)
+                .header(header::custom("Accept-Language", "Rust"))
+                .header(header::custom("Authorization", format!("Basic {}", token)))
+                .json(&model.form)
+                .expect("Serialization failed");
+
+            orders.perform_cmd(async {
+                let response = fetch(request).await.expect("HTTP request failed");
+                assert!(response.status().is_ok());
```

Comment:
Well assertions is not about tests, they are about invariants. In this case this means "I expect that status is ok, otherwise panic", i.e. [fail fast](https://en.wikipedia.org/wiki/Fail-fast)
As a side note, `assert` will still be present in the release build, compared to `debug_assert`.

Reply:
Essentially it's a placeholder akin to `.unwrap()` in example code.